### PR TITLE
Source context option in Far field, create_report and get_far_field_data methods

### PR DESCRIPTION
**Pull request by Samuelopez-ansys**

### CODE REVIEW

**Comment by Samuelopez-ansys** [comment]
@MaxJPRey I think there is an issue in pyaedt requirements. It is related to issue #1708

Could you please check it?

**Comment by MaxJPRey** [comment]
> @MaxJPRey I think there is an issue in pyaedt requirements. It is related to issue #1708
> 
> Could you please check it?

Sure @Samuelopez-ansys I will fix it.

File: pyaedt/modules/PostProcessor.py [hunk 1]
```diff
@@ -307,6 +307,8 @@ def far_field(self, expressions=None, setup_name=None, sphere_name=None):
             Setup Name.
         sphere_name : str, optional
             Name of the sphere on which create the far field.
+        source_context : str, optional
+            Name of the active source on which create the far field.
```
Comment (PipKat) [inline pyaedt/modules/PostProcessor.py:311]:
```suggestion
            Name of the active source to create the far field on.
```

File: pyaedt/modules/PostProcessor.py [hunk 2]
```diff
@@ -307,6 +307,8 @@ def far_field(self, expressions=None, setup_name=None, sphere_name=None):
             Setup Name.
         sphere_name : str, optional
             Name of the sphere on which create the far field.
```
Comment (PipKat) [inline pyaedt/modules/PostProcessor.py:309]:
```suggestion
            Name of the sphere to create the far field on.
```

File: pyaedt/modules/PostProcessor.py [hunk 3]
```diff
@@ -296,7 +296,7 @@ def rl_fields(self, expressions=None, setup_name=None, polyline=None):
         return
 
     @pyaedt_function_handler()
-    def far_field(self, expressions=None, setup_name=None, sphere_name=None):
+    def far_field(self, expressions=None, setup_name=None, sphere_name=None, source_context=None):
         """Create a Field Report object.
```
Comment (PipKat) [inline pyaedt/modules/PostProcessor.py:300]: Perhaps this should be "Create a far field object." The terminology here should be consistent with what is used later.

**Comment by Samuelopez-ansys** [comment]
@maxcapodi78 Could you help me with the IPython? I don't find the issue.

File: pyaedt/modules/PostProcessor.py [hunk 4]
```diff
@@ -306,7 +306,9 @@ def far_field(self, expressions=None, setup_name=None, sphere_name=None):
         setup_name : str, optional
             Setup Name.
```
Comment (PipKat) [inline pyaedt/modules/PostProcessor.py:307]:
```suggestion
            Setup name.
```

**Pull request merged**